lib/queue_classic/worker.rb

```diff
@@ -161,5 +163,15 @@ def setup_queues(adapter, queue, queues, top_bound)
       end
     end
 
+    def clone_args(job)
+      job[:args].map do |arg|
```

Comment:
Why is this a map? Could you just do (maybe I am being naive here!) `jobs[:args] = jobs[:args].clone`?